controllers/helmrepository_controller_test.go

```diff
@@ -127,6 +127,30 @@ func TestHelmRepositoryReconciler_Reconcile(t *testing.T) {
 		return obj.Status.LastHandledReconcileAt == "now"
 	}, timeout).Should(BeTrue())
 
+	// Switch to a OCI helm repository type
```

Comment:
This should probably be a separate test case, instead of one thing that attempts to cover all.